cli/src/pcluster/config/update_policy.py

```diff
@@ -125,6 +134,54 @@ def is_slurm_queues_change(change):
     return any(path.startswith("SlurmQueues[") for path in change.path)
 
 
+def extract_type_and_name_from_path(path):
+    obj_type = re.search(".+(?=\\[)", path).group(0)
+    obj_name = re.search("(?<=\\[).+(?=\\])", path).group(0)
+    return obj_type, obj_name
+
+
+def get_q_from_config(change, config):
```

Comment:
Let's add a sample string of the `change` to better understand how the regex parsing?/matching is happening.

Reply:
will do!